src/scratch/blocks/Advanced/Functions/procedures_callreturn.js

```diff
@@ -20,7 +25,15 @@ Blockly.Blocks.procedures_callreturn = {
             colour         : Blockly.Colours.BinaryProcedures.colour,
             colourSecondary: Blockly.Colours.BinaryProcedures.colourSecondary,
             colourTertiary : Blockly.Colours.BinaryProcedures.colourTertiary,
-        });
+            tooltip        : translate('Function with return value'),
+            category       : translate('functions'),
```

Comment:
do we need to translate this?

Reply:
you're right actually. Doesn't need to be translate